### ISSUE TITLE
How are the tests supposed to be run?

### ISSUE THREAD

**Issue by alexanderwallin** [audiojs/audio-context#12]
Hey!

I needed to verify the fix in #11 and wondered about the testing process. I installed browserify and testling and created a script:

```json
"test-browser": "browserify test.js | testling -u"
```

I then opened the provided URL in Firefox.

Is there a smoother way of running the browser tests?

**Comment by juliangruber** [comment]
```js
"test-browser": "browserify test.js | tape-run"
```

..is fully automated

**Comment by alexanderwallin** [comment]
Thanks @juliangruber. Is there a point in keeping `"test": "node test.js"`, seeing it won't run any of the tests?

**Comment by juliangruber** [comment]
you're right, we should remove that too

**Comment by alexanderwallin** [comment]
Alright, I'll see to it.

**Comment by dfcreative** [comment]
Fixed by #13
